packages/cads-validation/lib/component.validation.scss

```diff
@@ -7,21 +7,28 @@
 // component config
 //
 $validation-border-color: $extended--redPrimary;
-$validation-message-margin-top: $spacing--xs;
+$validation-message-margin-top: $spacing--sm;
 
 // component block
 //
 input:invalid {
   border-color: $validation-border-color;
 }
-.input--error{
-  color: $validation-border-color;
-  font-weight: normal;
-  margin-top: $validation-message-margin-top;
+.c-validation {
+  &--error {
+    color: $validation-border-color;
+    font-weight: normal;
+    margin-top: $validation-message-margin-top;
+    display: none;
 
-  //need to add the warning icon in here once the icons have been pushed
-  // &::before{
-  //     content:url('/assets/images/warning.svg');
-  //     margin-right: 5px;
-  // }
-}
\ No newline at end of file
+    &.active {
```

Comment:
@btomy the `error` class is spot on, let's do a similar thing with the `active` class too.
```
&--active { ... }
```
Obvs need a markup change too. :)